content/api/_index.en.md

```diff
@@ -1,31 +1,55 @@
+
+Instances are also a little bit like containers for all other documents; forms and unstructured data collected during an interaction between an end user and the application owner. The number of documents and files being produced will depend on the app. 
+
+In a simple app, an end user will fill in a form and submit it to the application owner. This would typically produce a one way interaction where only one or two documents are collected in the instance. In more advanced apps the number of documents and files would extend to multiple forms, attachments, and feedback or validation messages from the application owner. 
+
+An instance can't reference data directly, but will hold a list of data elements with more information about collected data.
+
+### DataElement
+
+The DataElement model is a metadata document for a specific form or binary file. The most important aspects of this document is that it holds information about where the actual data is being stored, and how the data is being used by the application. 
+
+Most apps will automatically create a data element to represent the form being filled out by the user. More advanced apps will also require the user to upload attachments or to fill in multiple forms.
+
+An instance can have many data elements, but each data element can not reference more than one data file.
+
 ## The APIs
 
-The new solution will have multiple APIs, but they can be divided in two groups. The app APIs and the Platform APIs.
+The Altinn 3 solution has multiple APIs, but they can be divided in two groups. The app APIs and the Platform APIs.
 
 ### App API
 
 The application API is an API that provides access to specific instances of a specific app. The API provides features for working with data elements while keeping the metadata document for the instance and its data elements updated. The instance level endpoints revolves around moving an instance through its defined process and controlling some instance level settings.
 
 Metadata for an app is the second job of the app API. There are endpoint that give access to the metadata of the app itself, its data types and process description.
 
-Every app will expose almost identical endpoints and functionality. External parties should need only one client implementation across all app APIs. Technically there is nothing preventing an application owner from adding or making changes to the API, but in those cases it is probably an application with a different process flow. In these cases, the application might require some special handling, and additional documentation should be provided by the application owner.
+Every app will expose almost identical endpoints and functionality. External parties should need only one client implementation across all app APIs. 
```

Comment:
It's possible to extend the API with custom controllers, should this be reflected here?

Reply:
Updated with a new text